scss/mixins/_display.scss

```
@@ -0,0 +1,12 @@
+@mixin dFlex($just: space-between, $align: center, $gap: 0) {
+  display: flex;
+  justify-content: $just;
+  align-items: $align;
+  gap: $gap;
+}
+
+@mixin flexCol($just: space-between, $align: center, $gap: 0) {
```

Đặt tên function, mixin,... sao cho đồng bộ với nhau
```suggestion
@mixin dFlexCol($just: space-between, $align: center, $gap: 0) {
```